lms/djangoapps/bulk_email/tests/test_tasks.py

```diff
@@ -104,15 +104,9 @@ def _create_input_entry(self, course_id=None):
         return instructor_task
 
     def _run_task_with_mock_celery(self, task_class, entry_id, task_id):
-        """Submit a task and mock how celery provides a current_task."""
-        mock_current_task = Mock()
-        mock_current_task.max_retries = settings.BULK_EMAIL_MAX_RETRIES
-        mock_current_task.default_retry_delay = settings.BULK_EMAIL_DEFAULT_RETRY_DELAY
+        """Mock was not needed for some tests, testing to see if it's needed at all."""
```

Comment:
I didn't want to change too many unrelated things in this PR so I'm leaving this function in for now, I'll have a followup PR to replace the function with just the call to `task_class.apply()` in all tests that use this function.